packages/typedoc-plugin-appium/lib/model/reflection/utils.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Adapted from `@knodes/typedoc-pluginutils`
+ * @see https://github.com/knodescommunity/typedoc-plugins
+ * Copyright (c) 2022 KnodesCommunity
+ * Licensed MIT
+ * @see https://github.com/KnodesCommunity/typedoc-plugins/blob/05717565fae14357b1c4be8122f3156e1d46d332/LICENSE
+ */
+
+import {ReflectionKind} from 'typedoc';
+
+const getHigherBitMask = () =>
```

Comment:
should the result be memozied?

Reply:
doesn't need to be. it should only be called like twice

Reply:
it's anyway always the same result. I don't see a reason to calculate it more than once